apps/sms/views/conversation/js/conversation.js

```diff
@@ -747,6 +752,7 @@ var ConversationView = {
     // TODO add the activity/forward/draft stuff here
     // instead of in afterEnter: Bug 1010223
 
+    // TODO: Will need to replace with Conversation service method.
```

Comment:
I don't think we need currentId in the service ? currentId is really a "view" characteristic: which view is currently displayed in this window.

I think the id is passed through the beforeEnter method, and can be kept locally in the view. I'm fine if we have another object that can hold the view's local model (including the id then) if we need to access it in a separate object, but we can also pass it as parameter instead when we need it.

I see this is used:
* in conversation.js -- ok, we remove it in this patch
* in utils.js/closeNotificationsForThread -- I see we always pass the argument so we can remove the code that uses currentId
* activity_handler.js -- we can replace by Navigation.isCurrentPanel
* information.js -- same, and I even think I do it in my navigation patch